### ISSUE TITLE
[wavereform] No tests. (Trac #1596)

### ISSUE THREAD

**Issue by icecube-trac** [icecube-trac/tix2#1588]
It's one of the few projects with 0% line coverage in the test itself.  It's a false report because 'wavereform/private/test' doesn't exist.  This is used in production and should have some tests.  Please write at least one for IceRec V5 and push the ticket to the next milestone.

Migrated from https://code.icecube.wisc.edu/ticket/1596
```json
{
    "status": "closed", 
    "changetime": "2019-02-13T14:11:57", 
    "description": "It's one of the few projects with 0% line coverage in the test itself.  It's a false report because 'wavereform/private/test' doesn't exist.  This is used in production and should have some tests.  Please write at least one for IceRec V5 and push the ticket to the next milestone.", 
    "reporter": "olivas", 
    "cc": "", 
    "resolution": "fixed", 
    "_ts": "1550067117911749", 
    "component": "combo reconstruction", 
    "summary": "[wavereform] No tests.", 
    "priority": "major", 
    "keywords": "", 
    "time": "2016-03-18T23:42:38", 
    "milestone": "", 
    "owner": "jbraun", 
    "type": "task"
}
```

**Comment by icecube-trac** [comment]
Trac update at 20160328T20:26:33: 
* jbraun changed resolution from "" to "fixed"
* jbraun commented:

> In changeset 143801:
> 
> ```text
> #!CommitTicketReference repository="IceCube" revision="143801"
> Fix CMakeLists.txt to actually run the test.  Closes #1588.
> ```


* jbraun changed status from "new" to "closed"

**Comment by icecube-trac** [comment]
Trac update at 20190213T14:11:57: 
* olivas changed milestone from "IceRec V5" to ""
* olivas commented:

> Milestone IceRec V5 deleted
